src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -569,6 +578,14 @@ private List<LaunchCommand> GetInitializeCommands()
                 }
             }
 
+            if (MICommandFactory.SupportsChildProcessDebugging())
+            {
+                if (_launchOptions.DebugChildProcesses)
+                {
+                    _childProcessHandler = new DebugUnixChild(this, this._launchOptions);  // TODO: let the user enable/disable this functionality
```

Comment:
That comment is referring to giving the user the ability to turn off and on the catching of forked children during a debugging session. Currently the user can only catch all children or no children, there is no finer control.


_____________________________
From: Gregg Miskelly <notifications@github.com<mailto:notifications@github.com>>
Sent: Monday, May 23, 2016 9:23 PM
Subject: Re: [Microsoft/MIEngine] Child2 (#329)
To: Microsoft/MIEngine <miengine@noreply.github.com<mailto:miengine@noreply.github.com>>
Cc: Paul Maybee <paulmay@microsoft.com<mailto:paulmay@microsoft.com>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>



In src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs<https://github.com/Microsoft/MIEngine/pull/329#discussion_r64326721>:

> @@ -569,6 +578,14 @@ private List<LaunchCommand> GetInitializeCommands()>                  }>              }>  > +            if (MICommandFactory.SupportsChildProcessDebugging())> +            {> +                if (_launchOptions.DebugChildProcesses)> +                {> +                    _childProcessHandler = new DebugUnixChild(this, this._launchOptions);  // TODO: let the user enable/disable this functionality

Should this TODO comment be removed now?

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/MIEngine/pull/329/files/1e0e5486fa6440bcea7c7bc67e0da514d5f2641d#r64326721>